modules/nw-rfhe-creating_bmc_event_sub.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * monitoring/using-rfhe.adoc
+
+:_content-type: PROCEDURE
+[id="nw-rfhe-creating_bmc_event_sub_{context}"]
+= Subscribing to Redfish hardware events
+
+You can configure the baseboard management controller (BMC) to send Redfish hardware events to subscribed applications running in {product-title} cluster. Example Redfish events include an increase in device temperature, or removal of a device. Applications subscribe to Redfish hardware events using a REST API.
+
+[IMPORTANT]
+====
+You can only create a `BMCEventSubscription` custom resource (CR) for physical hardware that supports Redfish and has a vendor interface set to `redfish` or `idrac-redfish`.
+====
+
+[NOTE]
+====
+Use the `BMCEventSubscription` CR to subscribe to predefined Redfish events. The Redfish standard does not provide an option to create specific alerts and thresholds. For example, to receive an alert event when an enclosure's temperature exceeds 40° Celsius, you must manually configure the event according to the vendor's recommendations.
+====
+
+Perform the following procedure to subscribe to Redfish hardware events for the node using a `BMCEventSubscription` CR.
+
+.Prerequisites
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Get the user name and password for the BMC.
+* Deploy a bare-metal node with Redfish hardware in your cluster, and enable Redfish events on the BMC.
```

Comment:
Flagging this sentence for inconsistent use of "bare-metal". In other parts of Redfish doc you have used "baremetal".